### PR TITLE
Type-Checking Static Members of a Class

### PR DESCRIPTION
**Pull request by ajvincent**
This is something that was non-obvious to me at first glance.  I figured writing down a simple example for others would not hurt.

### CODE REVIEW

**Comment by ajvincent** [comment]
@microsoft-github-policy-service agree

**Comment by ajvincent** [comment]
I'm thinking about moving this down to the next section, "Static blocks in classes", as an illustrative example which fits better there.

```typescript
interface VehicleType {
  manufacturer: string
}

interface VehicleTypeStatic {
  readonly wheelCount: number;
}

class Vehicle implements VehicleType 
{
  static {
    const temp: VehicleTypeStatic = Vehicle;
    void(temp);
  }

  readonly manufacturer: string;

  constructor(maker: string) {
    this.manufacturer = maker;
  }

  static readonly wheelCount = 4;
}
```

I'm wondering what people think of this alternative.

File: packages/documentation/copy/en/handbook-v2/Classes.md [hunk 1]
```diff
@@ -909,6 +909,38 @@ const MyHelperObject = {
 };
 ```
 
+### Type-Checking Static Members of a Class
+
+Under the hood, each class is an object (really a function) with a prototype property.  You can define a type for the class's static members, which will be properties of the class (not instances of it).
```
Comment (jakebailey) [inline packages/documentation/copy/en/handbook-v2/Classes.md:914]: I'm not totally sure that this section is worth including, at least not as-is.

Are you hoping to have an example that shows how to work around the lack of `class Vehicle extends static VehicleTypeStatic`? There's probably a simpler way to do that, if so; I'd think we'd want to be pretty targeted about showing the difference between the the "value" of the class and the "type" of a class instance. Not sure if this doc already includes some of that language, though.

**Review by jakebailey — CHANGES_REQUESTED** [review]

**Comment by ajvincent** [comment]
I may not be able to respond in a timely manner.  I have a new job which takes up a lot more of my time than I've been used to.  If anyone wants to close this pull request, I have no objection.

**Pull request closed without merging**